node/bft/src/helpers/dag.rs

```diff
@@ -22,9 +22,12 @@ use snarkvm::{
 use indexmap::IndexSet;
 use std::collections::{BTreeMap, HashMap};
 
+/// Maintains an acyclic directed graph (DAG) of batches, from which we build a totally-ordered blockchain.
```

Comment:
```suggestion
/// Maintains a directed acyclic graph (DAG) of batches, from which we build a totally-ordered blockchain.
```
Minor nit to match the terminology with the acronym